dex/price-discovery/README.md

```diff
@@ -2,32 +2,33 @@
 
```

Comment:
This readme is a bit out of date with the current design, but then again, the new implementation is on another branch, so this can stay as it is for now.